hadoop-mapreduce/src/test/java/org/apache/accumulo/hadoop/its/mapreduce/AccumuloInputFormatIT.java

```diff
@@ -148,22 +151,21 @@ public void testGetSplits() throws Exception {
       // overlapping ranges
       ranges.add(new Range(String.format("%09d", i), String.format("%09d", i + 2)));
     AccumuloInputFormat.configure().clientProperties(getClientInfo().getProperties()).table(table)
-        .auths(Authorizations.EMPTY).ranges(ranges).offlineScan().store(job);
+        .auths(Authorizations.EMPTY).ranges(ranges).offlineScan(true).store(job);
     splits = inputFormat.getSplits(job);
     assertEquals(2, splits.size());
 
     AccumuloInputFormat.configure().clientProperties(getClientInfo().getProperties()).table(table)
-        .auths(Authorizations.EMPTY).disableAutoAdjustRanges().offlineScan().store(job);
+        .auths(Authorizations.EMPTY).autoAdjustRanges(true).offlineScan(true).store(job);
     splits = inputFormat.getSplits(job);
     assertEquals(ranges.size(), splits.size());
 
     // BatchScan not available for offline scans
     AccumuloInputFormat.configure().clientProperties(getClientInfo().getProperties()).table(table)
-        .auths(Authorizations.EMPTY).batchScan().store(job);
-    try {
-      inputFormat.getSplits(job);
-      fail("An exception should have been thrown");
-    } catch (IOException e) {}
+        .auths(Authorizations.EMPTY).batchScan(true).store(job);
+
+    exception.expect(IllegalArgumentException.class);
+    inputFormat.getSplits(job);
```

Comment:
Could add an illegal arg exception to store also

Reply:
Yeah I like that.  I will move the checks to a method to share.

Reply:
Actually, I will do this in #749 since they are verified on a per table basis.